src/AppBundle/Resources/views/periodical/edit_contribution.html.twig

```diff
@@ -0,0 +1,16 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Periodical Contribution Edit{% endblock %}
+
+{% block body %}
+    <h1>Periodical Contribution Edit</h1>
+
+    {{ form_start(edit_form) }}
+    {{ form_widget(edit_form) }}
+    <div class='btn-group'>
+        <input type="submit" value="Update" class="btn btn-primary" />
+        <a href="{{ path('periodical_show_contributions', { 'id': contribution.publication.id }) }}" class="btn btn-warning">Cancel</a>
+    </div>
+    {{ form_end(edit_form) }}
+
+{% endblock %}
```

Comment:
And same comment about the blocks as for compilation/edit_contribution.html.twig